doc/news/DM-43815.mtaircompressor

```diff
@@ -0,0 +1 @@
+Added missign description entries to XML.
```

Comment:
`missign` -> `missing`. Also please always use an active voice so `Add` instead of `Added`.